pkgs/flarum/php-packages.nix

```diff
@@ -0,0 +1,1358 @@
+          url = "https://api.github.com/repos/symfony/translation/zipball/de237e59c5833422342be67402d487fbf50334ff";
+          sha256 = "10qc1fymfvchk161hykkbqpy5ndxh6kigjhjsz5qp968dqcrckj2";
+        };
+      };
+    };
+    "symfony/translation-contracts" = {
+      targetDir = "";
+      src = composerEnv.buildZipPackage {
+        name = "symfony-translation-contracts-136b19dd05cdf0709db6537d058bcab6dd6e2dbe";
+        src = fetchurl {
+          url = "https://api.github.com/repos/symfony/translation-contracts/zipball/136b19dd05cdf0709db6537d058bcab6dd6e2dbe";
+          sha256 = "1z1514i3gsxdisyayzh880i8rj954qim7c183cld91kvvqcqi7x0";
+        };
+      };
+    };
+    "symfony/yaml" = {
+      targetDir = "";
+      src = composerEnv.buildZipPackage {
+        name = "symfony-yaml-4cd2e3ea301aadd76a4172756296fe552fb45b0b";
+        src = fetchurl {
+          url = "https://api.github.com/repos/symfony/yaml/zipball/4cd2e3ea301aadd76a4172756296fe552fb45b0b";
+          sha256 = "18yirwiqbh11fwlpqypm0wlc0lnx7prgk68xrn607zcg6cxnfhiz";
+        };
+      };
+    };
+    "tijsverkoyen/css-to-inline-styles" = {
+      targetDir = "";
+      src = composerEnv.buildZipPackage {
+        name = "tijsverkoyen-css-to-inline-styles-c42125b83a4fa63b187fdf29f9c93cb7733da30c";
+        src = fetchurl {
+          url = "https://api.github.com/repos/tijsverkoyen/CssToInlineStyles/zipball/c42125b83a4fa63b187fdf29f9c93cb7733da30c";
+          sha256 = "0ckk04hwwz0fdkfr20i7xrhdjcnnw1b0liknbb81qyr1y4b7x3dd";
+        };
+      };
+    };
+    "voku/portable-ascii" = {
+      targetDir = "";
+      src = composerEnv.buildZipPackage {
+        name = "voku-portable-ascii-87337c91b9dfacee02452244ee14ab3c43bc485a";
+        src = fetchurl {
+          url = "https://api.github.com/repos/voku/portable-ascii/zipball/87337c91b9dfacee02452244ee14ab3c43bc485a";
+          sha256 = "1j2xpbv7xiwxwb6gfc3h6imc6xcbyb2jw3h8wgfnpvjl5yfbi4xb";
+        };
+      };
+    };
+    "webmozart/assert" = {
+      targetDir = "";
+      src = composerEnv.buildZipPackage {
+        name = "webmozart-assert-11cb2199493b2f8a3b53e7f19068fc6aac760991";
+        src = fetchurl {
+          url = "https://api.github.com/repos/webmozarts/assert/zipball/11cb2199493b2f8a3b53e7f19068fc6aac760991";
+          sha256 = "18qiza1ynwxpi6731jx1w5qsgw98prld1lgvfk54z92b1nc7psix";
+        };
+      };
+    };
+    "wikimedia/less.php" = {
+      targetDir = "";
+      src = composerEnv.buildZipPackage {
+        name = "wikimedia-less.php-0d5b30ba792bdbf8991a646fc9c30561b38a5559";
+        src = fetchurl {
+          url = "https://api.github.com/repos/wikimedia/less.php/zipball/0d5b30ba792bdbf8991a646fc9c30561b38a5559";
+          sha256 = "06f5z6g4pk6lvmakcsdwysyi6yknmvqk7p1krb0zra7hi41qwirr";
+        };
+      };
+    };
+  };
+  devPackages = {};
+in
+composerEnv.buildPackage {
+  inherit packages devPackages noDev;
+  name = "flarum-flarum";
+  src = composerEnv.filterSrc ./.;
```

Comment:
This line seems to be written assuming that this `nix` expression will be evaluated in the project repo, but because it's in ngipkgs, we only end up with `composer.lock` and the `vendor` directory

Reply:
We came across https://discourse.nixos.org/t/source-combinators/13331/8 and https://discourse.nixos.org/t/easy-source-filtering-with-file-sets/29117 but neither seem to be usable yet

Reply:
Yeah, this is what I feared. `composer2nix` makes a few assumptions here that we can't work around without ripping open the whole thing. @infinisil is still working on file set combinators, but we may get enough of it into Nixpkgs for you to use.